test/unit/event.js

```diff
@@ -2686,6 +2686,134 @@ test( "Inline event result is returned (#13993)", function() {
+	} );
+	jQuery( "#donor-input" ).on( "click", function( event ) {
+		assert.ok( true, "got a click event from the input" );
+		assert.ok( !event.isPropagationStopped(), "propagation says it's not stopped" );
+		assert.equal( event.type, "click", "make sure event type is correct" );
+		assert.equal( typeof event.originalEvent, "object", "make sure originalEvent exist" );
+	} );
+	jQuery( "#donor-input" ).on( "change", function( event ) {
+		assert.equal( typeof event.originalEvent, "object", "make sure originalEvent exist" );
+		assert.equal( event.type, "change", "make sure event type is correct" );
+		assert.ok( true, "got a change event from the input" );
+		event.stopPropagation();
+	} );
+	jQuery( "#donor-input" )[0].click();
+} );
+
+test( "originalEvent property for IE8", function( assert ) {
+	if ( !(/msie 8\.0/i.test( window.navigator.userAgent )) ) {
+		assert.expect( 1 );
+		assert.ok( true, "Assertions should run only in IE" );
+		return;
+	}
+
+	assert.expect( 12 );
+
+	var html = "<div id='donor-outer'>" +
+		"<form id='donor-form'>" +
+			"<input id='donor-input' type='radio' />" +
+		"</form>" +
+	"</div>";
+
+	jQuery( "#qunit-fixture" ).append( html );
+
+	jQuery( "#donor-outer" ).on( "change", function( event ) {
+		assert.ok( true, "click bubbled to outer div" );
+		assert.equal( event.originalEvent.type, "click", "make sure simulated event is a click" );
+		assert.equal( event.type, "change", "make sure event type is correct" );
+	} );
+
+	jQuery( "#donor-outer" ).on( "click", function( event ) {
+		assert.ok( true, "click bubbled to outer div" );
+		assert.equal( event.originalEvent.type, "click", "make sure originalEvent exist" );
+	} );
+	jQuery( "#donor-input" ).on( "click", function( event ) {
+		assert.ok( true, "got a click event from the input" );
+		assert.ok( !event.isPropagationStopped(), "propagation says it's not stopped" );
+		assert.equal( event.originalEvent.type, "click", "make sure originalEvent exist" );
+		assert.equal( event.type, "click", "make sure event type is correct" );
+	} );
+	jQuery( "#donor-input" ).on( "change", function( event ) {
+		assert.equal( event.originalEvent.type, "click", "make sure originalEvent exist" );
+		assert.equal( event.type, "change", "make sure event type is correct" );
+		assert.ok( true, "got a change event from the input" );
+	} );
+	jQuery( "#donor-input" )[0].click();
+} );
+
+test( "originalEvent property for Chrome, Safari and FF of simualted event", function( assert ) {
+	var userAgent = window.navigator.userAgent;
+
+	if ( !(/chrome/i.test( userAgent ) ||
+	       /firefox/i.test( userAgent ) ||
+	       /safari/i.test( userAgent ) ) ) {
+		assert.expect( 1 );
+		assert.ok( true, "Assertions should run only in Chrome, Safari and FF" );
+		return;
+	}
+
+	assert.expect( 4 );
+
+	var html = "<div id='donor-outer'>" +
+		"<form id='donor-form'>" +
+			"<input id='donor-input' type='radio' />" +
+		"</form>" +
+	"</div>";
```

Comment:
Use of `supportjQuery` would help you create an isolated env for you, that is cool, but that is a unit-test which means you testing smallest possible action and change should be atomic, so really, you would need to use this entity only when you create an environment through the thing you test - only in if you expect that your changes could affect other parts of library.

And even if your change has effect on assertion output it probably be false negative then false positive which would additionally indicate that something is wrong.

In other words, if you test `jQuery.get` you could use `jQuery#html` in your tests, because your changes on `jQuery.get` shouldn't influence entirely different module (at least in most casez) since it should be atomic and even if it does, then `jQuery.get` assertions would fail then give right output. But if you need to use `jQuery.get` to create fixtures for `jQuery.get` assertions then you should use `supportjQuery`.

I'm not sure what is right thing to do, best action might be to go from the real-life example, although it seems `supportjQuery` is the most correct way. But then there is a consistency issue, we are not gonna rewrite 6 thousands tests because of it, do we?

Now, tests with wrong kinds of fixtures is another story, we probably should start rewriting at least some of them, since we need it to use more flexible testing tools like karma.